client/modules/IDE/components/SketchList.jsx

```diff
@@ -20,6 +21,13 @@ class SketchList extends React.Component {
     document.getElementById('sketchlist').focus();
   }
 
+  componentDidUpdate() {
+    if (this.props.sketches.length === 0) {
```

Comment:
The server now returns a 404 error when the user is not found. This means that the response is not being passed as a prop anymore so all I can do is check if the array is empty and toast that no sketches were found.